src/org/ensembl/healthcheck/testcase/variation/CheckChar.java

```diff
@@ -162,10 +161,10 @@ public String getUnsupportedChar( String input) {
         is_ok = false;
       }
       if (!is_ok) {
-        unsupportedChar = "" + input.charAt(i);
+        return Character.toString ((char) ascii_val);
```

Comment:
Suggest maybe not having space between toString and (
i.e. return Character.toString((char) ascii_val)